backend/metadata/databases/default/tables/public_Person.yaml

```diff
@@ -55,6 +55,7 @@ select_permissions:
   role: anonymous
 - permission:
     columns:
+    - AuthId
```

Comment:
Is there a special reason for this? I personally find it difficult to expose the secret keys of every user to the public.

Reply:
We need this `AuthId` the filter of Attendances: `Attendences(where: { Person: { AuthId: { _eq: $authId } } })`.
IMHO the `AuthId` isn't a secret.

Asked in passing: Is there a typo in the database `Attendences`?

Reply:
Yep, seems it should be `Attendances`.

Reply:
The correction is already part of the Backend Changes in issue #49